bin/heat-flow-module-L0/3_thermal_conductivity_from_geotherms.ipynb

```diff
@@ -0,0 +1,432 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    # Create a function to convert depth to pressure
</pre><p>Add function documentation here? I realize this is just a single line, but perhaps best to be consisent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Aa oo, forgot to add those! Thank you for catching them. :)